roboteam_ai/src/skills/Skill.cpp

```diff
@@ -38,28 +28,31 @@ Skill::Status Skill::update() {
     updateRobot();
     ball = World::getBall(); // update ball position
     if (! robot) return Status::Failure;
+    if (! ball) return Status::Waiting;
     return onUpdate();
 }
 
 void Skill::initialize() {
     robot = getRobotFromProperties(properties);
     ball = World::getBall();
-    if (!robot) return;
+    if (! robot) return;
+    if (! ball) return;
```

Comment:
now skills don't work if there is no ball! this can be annoying

Reply:
More annoying if the skill segfaults when there is no ball ;)

Reply:
You should then be able to say if a skilll can /cannot run without a ball.